### ISSUE TITLE
Plans

### ISSUE THREAD

**Issue by seanamor** [ata4/dragon-mounts#21]
Are there plans to update to 1.8.9? I and all sorts of users who are indeed obsessed with your work, are all waiting to hear :D This is an amazing concept, and I trully hope you dont lose interest. I can provide art, texturing, model design, and all sorts of other graphic work if this motivates you by any means.

Thanks for your time!

RealmofThaumtology.enjin.com
SurvivalPlanetEarth.enjin.com
Skype: Sean.Amor.Mc
Gmail: Sean.Amor.Mc
IGN: Sean_Amor

**Comment by ata4** [comment]
Yes, I've been working on a port for a while. Seems to work pretty good so far, but I can't give an ETA yet. I need to be sure this time that all the features are still working correctly.

**Comment by seanamor** [comment]
Excellent, just glad to see its not going to stop where it is, as I said before, if you need graphic help im all yours. No charge obviously.
